### PR TITLE
COMMERCE-11447 use right serviceContext

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://issues.liferay.com/browse/COMMERCE-11447

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Review by luca-pellizzon — APPROVED** [review]

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:commerce-functional

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/util/v1_0/AttachmentUtil.java [hunk 1]
```diff
@@ -276,16 +277,21 @@ public static CPAttachmentFileEntry addOrUpdateCPAttachmentFileEntry(
 			ServiceContext serviceContext)
 		throws Exception {
 
-		ServiceContext cloneServiceContext =
+		ServiceContext dlFileEntrycloneServiceContext =
```
Comment (alessiorendina) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/util/v1_0/AttachmentUtil.java:280]: Typo here @andrea-ale-sbarra 

it should be `dlFileEntryCloneServiceContext`

**Comment by andrea-ale-sbarra** [comment]
ci:test:commerce-functional

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
QA passed. Add automation and resent on https://github.com/liferay-commerce/liferay-portal/pull/4112.

**Comment by jaydawu** [comment]
ci:close

**Pull request closed without merging**